### ISSUE TITLE
Unable to upgrade to 2.0.0-beta1 - require illuminate/support ^10.0

### ISSUE THREAD

**Issue by ajtrichards** [tonysm/turbo-laravel#132]
I'm trying to get the latest version 2.0.0-beta1 installed in my environment but i'm having the following error;

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires hotwired-laravel/turbo-laravel ^2.0.0-beta1 -> satisfiable by hotwired-laravel/turbo-laravel[2.0.0-beta1, 2.x-dev].
    - hotwired-laravel/turbo-laravel[2.0.0-beta1, ..., 2.x-dev] require illuminate/support ^10.0 -> found illuminate/support[v10.0.0, ..., 10.x-dev] but these were not loaded, likely because it conflicts with another require.
 ```

This is currently what I have in my `composer.json` file;

```
{
    "name": "laravel/laravel",
    "description": "The Laravel Framework.",
    "keywords": [
        "framework",
        "laravel"
    ],
    "license": "MIT",
    "type": "project",
    "require": {
        "php": "^8.1",
        "ext-json": "*",
        "24slides/laravel-saml2": "^2.3",
        "doctrine/dbal": "^2.8",
        "hubspot/api-client": "^8.4",
        "laravel/framework": "^9.0",
        "laravel/helpers": "^1.6",
        "laravel/legacy-factories": "^1.3",
        "laravel/tinker": "^2.0",
        "laravel/ui": "^4.2",
        "league/csv": "^9.0",
        "league/flysystem-aws-s3-v3": "^3.0",
        "predis/predis": "^1.1",
        "spatie/ignition": "^1.0",
        "spatie/laravel-permission": "^6.1",
        "zendesk/zendesk_api_client_php": "^2.2"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.9",
        "fakerphp/faker": "^1.20",
        "filp/whoops": "^2.0",
        "mockery/mockery": "^1.0",
        "nunomaduro/collision": "^6.0",
        "phpunit/phpunit": "^9.0",
        "spatie/laravel-ray": "^1.33"
    },
    "autoload": {
        "classmap": [
            "database/seeds",
            "database/factories"
        ],
        "psr-4": {
            "App\\": "app/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "scripts": {
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate"
        ],
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover"
        ]
    },
    "config": {
        "preferred-install": "dist",
        "sort-packages": true,
        "optimize-autoloader": true,
        "allow-plugins": {
            "kylekatarnls/update-helper": true
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

I've ran `composer update` and that hasn't solved. Not sure what else to try at the moment. Does it need the latest version of Laravel?

**Comment by tonysm** [comment]
@ajtrichards can you update to Laravel 10? v2 requires PHP 8.2 and Laravel 10.

**Comment by ajtrichards** [comment]
Hey @tonysm - thanks - that solved it! 8.2 & Laravel 10 :)
